test/fixtures/embeds.html

```diff
@@ -25,14 +25,14 @@ <h2 id="image-style-embeds">Image-Style Embeds</h2>
     src="https://adobeioruntime.net/api/v1/web/helix/helix-services/embed@v1/https://www.youtube.com/watch?v=KOxbO0EI4MA">
   </esi:include>
   <esi:remove>
-    <p><img src="https://www.youtube.com/watch?v=KOxbO0EI4MA"></p>
+    <p><img src="https://www.youtube.com/watch?v=KOxbO0EI4MA" alt=""></p>
```

Comment:
**Note**: The new markdown parser now generates empty string values for the `alt` properties in mdast. they used to be `null`.  I think this doesn't affect the semantics of the HTML.

Reply:
It does affect the semantics, empty `alt` means screen readers will skip the image entirely because it's just decoration, missing `alt` attributes will cause the `src` attribute to be read. There may be an existing issue in remark already.

Reply:
which is actually a good thing:
https://www.w3.org/WAI/tutorials/images/decorative/

without having a default to "", there is no way to create a decorative image in markdown.
I think this is a good behaviour - however, I filed a bug anyways: https://github.com/micromark/micromark/issues/32

Reply:
Thanks for filing the issue, I agree with @wooorm's trade-offs.